docs/docs/development/new-shield.md

```diff
@@ -435,26 +435,31 @@ left_encoder: encoder_left {
         label = "LEFT_ENCODER";
         a-gpios = <PIN_A (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
         b-gpios = <PIN_B (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
-        resolution = <4>;
+        steps = <20>;
```

Comment:
```suggestion
        steps = <80>;
```
How about changing the example here to a common default?

Reply:
Also, can you fix the indentation so the first line is consistent with the other lines?